.github/workflows/docker.yml

```
@@ -48,9 +48,21 @@ jobs:
         file: ./Dockerfile-16.04
         tags: yourbase/yb_ubuntu:16.04
         push: ${{ github.event_name == 'push' }}
+    - name: Build 16.04 yb-user image
+      uses: docker/build-push-action@v2
+      with:
+        file: ./Dockerfile-yb-user-16.04
+        tags: yourbase/yb_ubuntu_yb-user:16.04
+        push: ${{ github.event_name == 'push' }}
     - name: Build 18.04 image
       uses: docker/build-push-action@v2
       with:
         file: ./Dockerfile-18.04
         tags: yourbase/yb_ubuntu:18.04
         push: ${{ github.event_name == 'push' }}
+    - name: Build 18.04 yb-user image
+      uses: docker/build-push-action@v2
+      with:
+        file: ./Dockerfile-yb-user-18.04
+        tags: yourbase/yb_ubuntu_yb-user:18.04
```

(See above for rationale)

```suggestion
        tags: yourbase/yb_ubuntu_nonroot:18.04
```